ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml

```diff
@@ -27,6 +27,21 @@ tests:
     make unit-test
   container:
     from: src
+- as: e2e-presubmit
+  steps:
+    cluster_profile: aws-qe
+    env:
+      OPENSHIFT_VERSION: "4.13"
+    pre:
```

Comment:
From the usage, it should be `test` rather than `pre`. `pre` are always the steps for preparation before testing.